appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/cmd/options/RuntimeOptions.java

```diff
@@ -111,29 +121,19 @@ public RuntimeOptions(String args[]) throws ValidationException {
                              }
                         }
                     }
-                    List<String> values = options.get(option);
-                    if (values == null) {
-                        values = new LinkedList<>();
-                        options.put(option, values);
-                    }
-                    values.add(value);
+                    options.add(new AbstractMap.SimpleImmutableEntry<>(option, value));
                 } catch (IllegalArgumentException iae) {
-                    throw new ValidationException(MessageFormat.format(commandlogstrings.getString("notValidArgument"),arg));
+                    throw new ValidationException(MessageFormat.format(commandlogstrings.getString("notValidArgument"), arg));
                 } catch (IndexOutOfBoundsException ex) {
-                    throw new ValidationException(MessageFormat.format(commandlogstrings.getString("expectedArgument"),arg));
+                    throw new ValidationException(MessageFormat.format(commandlogstrings.getString("expectedArgument"), arg));
                 } catch (ValidationException ve) {
-                    throw new ValidationException(arg + " " + ve.getMessage(),ve);
+                    throw new ValidationException(arg + " " + ve.getMessage(), ve);
                 }
-            } else if (arg.endsWith(".war") || arg.endsWith(".ear") || arg.endsWith(".rar") || arg.endsWith(".jar")) {
+            } else if (arg.endsWith(".war") || arg.endsWith(".rar") || arg.endsWith(".jar")) {
```

Comment:
Oh, right, we actually don't support `.ear`.

Reply:
Yes, in micro edition ear does not supports